decompiler/Function/Warnings.h

```diff
@@ -80,14 +86,16 @@ class DecompWarnings {
 
     std::string print() const {
       switch (warning_kind) {
-        case Kind::GENERAL:
+        case Kind::GENERAL_WARN:
           return fmt::format("WARN: {}\n", message);
+        case Kind::GENERAL_FAIL:
+          return fmt::format("ERROR: {}\n", message);
         case Kind::EXPR_BUILD_FAILED:
-          return fmt::format("WARN: Expression building failed: {}\n", message);
+          return fmt::format("ERROR: Expression building failed: {}\n", message);
         case Kind::CFG_FAILED:
-          return fmt::format("WARN: CFG building failed: {}\n", message);
+          return fmt::format("ERROR: CFG building failed: {}\n", message);
         case Kind::TYPE_PROP_FAILED:
-          return fmt::format("WARN: Type Propagation failed: {}\n", message);
+          return fmt::format("ERROR: Type Propagation failed: {}\n", message);
```

Comment:
If you wanted, I think this might be simpler if there were just 3 types: INFO, WARN, ERROR